samples/schema-registry-integration/schema-registry-consumer-kafka/mvnw

```diff
@@ -0,0 +1,226 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
I believe we can pull the wrapper up to the `schema-registry-integration` folder and avoid triplicating this. I did that for the confluent sample.